packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -41,6 +41,8 @@ module.exports = (resolve, rootDir, isEjecting) => {
         : resolve('config/jest/babelTransform.js'),
       '^.+\\.css$': resolve('config/jest/cssTransform.js'),
       '^(?!.*\\.(js|jsx|css|json)$)': resolve('config/jest/fileTransform.js'),
+      // ZEAL: Adds support for parsing GraphQL files
+      '^.+\\.(gql|graphql)$': require.resolve('jest-transform-graphql'),
```

Comment:
I think this should be `resolve` instead of `require.resolve`; `resolve` is an injected dependency, it appears.